library/src/window_impl.cxx

```diff
@@ -215,6 +317,17 @@ void window_impl::Initialize(bool withColoring)
   this->Internals->WithColoring = withColoring;
   this->Internals->Renderer->Initialize(this->Internals->Options.getAsString("scene.up-direction"));
   this->Internals->Initialized = true;
+
+#ifdef _WIN32
+  HWND hwnd = static_cast<HWND>(this->Internals->RenWin->GetGenericWindowId());
+
+  BOOL useDarkMode = this->Internals->IsWindowsInDarkMode();
+
+  if (this->Internals->IsWindowsBuildNumberOrGreater(22000))
+  {
+    DwmSetWindowAttribute(hwnd, DWMWA_USE_IMMERSIVE_DARK_MODE, &useDarkMode, sizeof(useDarkMode));
+  }
```

Comment:
```suggestion
  if (this->Internals->IsWindowsBuildNumberOrGreater(22000))
  {
    const HWND hwnd = static_cast<HWND>(this->Internals->RenWin->GetGenericWindowId());
    const bool useDarkMode = this->Internals->IsWindowsInDarkMode();
    DwmSetWindowAttribute(hwnd, DWMWA_USE_IMMERSIVE_DARK_MODE, &useDarkMode, sizeof(useDarkMode));
  }
```
these variables aren't needed outside the `if` block

Reply:
`BOOL` is a windows specific type. I am not sure if its ok to change it to bool.